README.md

```diff
@@ -186,8 +192,9 @@ render(
 
 Two callbacks are available on `RemoteFramesProvider`:
 
-- `onFrameAdded`: gets call whenever another frame is added to the stack
-- `onNoFrames`: gets call whenever all frames are removed from the stack
+- `onFrameAdded`: gets called whenever another frame is added to the stack
```

Comment:
perhaps better wording for "frame" is "RemoteFrame instance"

Reply:
I used the existing wording as ref

Reply:
yeah true :D tried to push improvement on all ;)